sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -122,7 +122,7 @@ abstract class SessionCatalogSuite extends AnalysisTest with Eventually {
   }
 
   test("create table with default columns") {
-    withBasicCatalog { catalog =>
+    if (!isHiveExternalCatalog) withBasicCatalog { catalog =>
```

Comment:
What happens if we remove `if (!isHiveExternalCatalog)`? What is the test failure in this case?